src/server/scripts/Custom/SkipDK.cpp

```diff
@@ -1,4 +1,4 @@
-/*
+﻿/*
```

Comment:
is this dealing with unicode adjustments? i usually save the files as utf-8 unsigned to ensure language compatibility with the quick gossip hard code translations. I am not really a fan of diluting the db and only make the adjustments to sql as bare basic needed.

Reply:
Might have been visual studio on save...

Reply:
New commit should fix the encoding back to UTF8